stock_inventory_revaluation/models/product.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
+
+
+class ProductCategory(models.Model):
+
+    _inherit = 'product.category'
+
+    property_inventory_revaluation_increase_account_categ = fields.Many2one(
+        'account.account', string='Valuation Increase Account',
+        company_dependent=True,
+        help="Define the G/L accounts to be used as the balancing account in "
```

Comment:
may be nice to replace G/L with plain words, for those like me who are not experts and may need the feature.